test/filters/unit_tests.filter

```diff
@@ -5,8 +5,44 @@
 ## why the filter is required and create an associated tracking issue.
 ##
 
+# These tests crash due to TestingBraveBrowserProcess::process_misc_metrics()
```

Comment:
didn't we just fix this to re-enable thing when process_misc_metrics is null?

Reply:
Yes, this should be fixed by https://github.com/brave/brave-core/pull/20433, I just need to rebase and remove these tests from the filter.